front/src/pages/index.jsx

```diff
@@ -0,0 +1,36 @@
+import CategoryCard from "@/components/CategoryCard"
+import Page from "@/components/Page"
+import { getCategories } from "@/utils/services/categoryServices"
+import { serverSideTranslations } from "next-i18next/serverSideTranslations"
+import { useTranslation } from "next-i18next"
+
+export const getServerSideProps = async ({ locale }) => {
+  const { data: categories } = await getCategories()
+
+  return {
+    props: {
+      categories,
+      ...(await serverSideTranslations(locale, ["common"]))
+    }
+  }
+}
+const Home = props => {
+  const { t } = useTranslation()
+  const { categories } = props
+
+  return (
+    <Page className="h-screen text-secondary">
+      <div className="w-full h-1/4 sm:h-1/3 bg-[url('https://picsum.photos/1800/350')] bg-center flex items-center justify-center">
```

Comment:
Don't put a lorem picsum url but try to have that will be the same all the time.

Reply:
Done!